drivers/display/display_ist3931.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024 Shen Xuyang  <shenxuyang@shlinyuantech.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef IST3931_DISPLAY_DRIVER_H__
+#define IST3931_DISPLAY_DRIVER_H__
+
+#include <zephyr/kernel.h>
```

Comment:
Why are including the kernel header here? If the driver needs this header, include it in the driver C file.